### PR TITLE
Added call_stack_contains_unbacked field

### PR DESCRIPTION
**Pull request by bit-envoy**
## Change Summary

Schema changes related to https://github.com/elastic/endpoint-dev/pull/11979

## Release Target

8.6.0

### CODE REVIEW

**Comment by jdu2600** [comment]
I believe that you just need to add it under `thread.Ext` something like this -
https://github.com/elastic/endpoint-package/blob/c2d8f902c45184eb24b68a4158bc5e3cbba0cc22/custom_schemas/custom_process.yml#L104-L106

And these earlier lines handle making reusable and it valid at locations like `Target.process.parent.thread.Ext` -
https://github.com/elastic/endpoint-package/blob/c2d8f902c45184eb24b68a4158bc5e3cbba0cc22/custom_schemas/custom_process.yml#L12-L16

**Comment by bit-envoy** [comment]
> I believe that you just need to add it under `thread.Ext` something like this -
> 
> https://github.com/elastic/endpoint-package/blob/c2d8f902c45184eb24b68a4158bc5e3cbba0cc22/custom_schemas/custom_process.yml#L104-L106
> 
> And these earlier lines handle making reusable and it valid at locations like `Target.process.parent.thread.Ext` -
> 
> https://github.com/elastic/endpoint-package/blob/c2d8f902c45184eb24b68a4158bc5e3cbba0cc22/custom_schemas/custom_process.yml#L12-L16

Fixed, thanks.

**Comment by kevinlog** [comment]
@bit-envoy 

Thanks for opening this PR!  There are a few things to make sure you do in order to get a green build on this.

1. You've added a new field to `custom_schemas`, but you need to make sure you add it to `custom_subsets`.  Based on the README [change here](https://github.com/elastic/endpoint-package/pull/308/files#diff-60d55abe6df4fb11a7072d19536395d005c98152b7359409efe16f21923fddf8R315), it looks like this is to be a new field in alerts.  You likely need to make a change to one of the `custom_subsets` [here](https://github.com/elastic/endpoint-package/tree/master/custom_subsets/elastic_endpoint/alerts)
2. After you make the changes, run `make` to ensure the same changes show up in `package/endpoint/data_stream`.  Your `make` will add fields to `alerts` `data_stream` I am assuming.  Be sure to commit these changes in your PR.
3. Add sample data for your change here, `package/endpoint/data_stream/<streamname>/sample_event.json`.  This helps us document the intent behind package changes.  In addition, after you update with sample data, CI will effectively use it as a unit test and check to see that the sample data you add matches the mapping changes you make.

I think [this recent change](https://github.com/elastic/endpoint-package/pull/307) has examples of what's described above.

Let me know of any questions you have

**Comment by bit-envoy** [comment]
Thanks @kevinlog for tips. I have updated custom_subsets, ran make successfully, but it seems make didn't add anything to package/endpoint/data_stream. What could be wrong?

File: custom_schemas/custom_process.yml [hunk 1]
```diff
@@ -95,6 +95,11 @@
       level: custom
       type: object
       description: Object for all custom defined fields to live in.
+      
+    - name: thread.Ext.call_stack_contains_unbacked
+      level: custom
+      type: boolean
+      description: Indicates if process has been created by parent that it's thread stack contains frames pointing outside image
```
Comment (gabriellandau) [inline custom_schemas/custom_process.yml:102]:
```suggestion
      description: Indicates whether the creating thread's stack contains frames pointing outside any known executable image.
```

**Comment by kevinlog** [comment]
@bit-envoy 

> I added changes manually. Is there any clear indication what files should be edited manually and what files should be generated?

I'll check out your PR and take a look to see if I see anything

**Review by gabriellandau — APPROVED** [review]

**Review by kevinlog — APPROVED** [review]
LGTM - thanks for the changes!

**Pull request merged**